### ISSUE TITLE
TO DO: Parse participants into Type, Name, Role, and Entity Affiliation into separate fields/values

### ISSUE THREAD

**Issue by TSoellinger** [LexPredict/nlrb_data#3]
Currently these are in a single string.

Current code .pop()'s the participant table so it would be better to do this by parsing the html object, rather than trying to pull the string apart in pandas given the variation. 

We can pull this apart and create columns for each element when parsing the html, OR add a ";" or similar delimiter between elements to easily str.split with pandas in post processing.

**Comment by mjbommar** [comment]
Resolved by commit ed0cbb2.  Build passing

**Comment by mjbommar** [comment]
```
>>> case_info = get_case("01-CA-104714")
>>> print(case_info["participants"].columns)
Index(['party_address', 'party_firm', 'party_name', 'party_phone',
       'party_role', 'party_type'],
      dtype='object')
```
